pgmpy/estimators/MmhcEstimator.py

```diff
@@ -177,7 +178,7 @@ def max_min_heuristic(X, Zs):
 
         # Find parents and children for each node
         neighbors = dict()
-        for node in nodes:
+        for node in tqdm(nodes):
```

Comment:
@jovalie Some users don't like to have progress bars, so we have tried to always have an argument `show_progress` in the function/method to specify whether to show progress or not. Also, we need to consider the global var `SHOW_PROGRESS` (https://github.com/pgmpy/pgmpy/blob/dev/pgmpy/global_vars.py#L40). In this case, I would instead not use the progress bar because I don't think this is computationally expensive enough to take substantial time to run.

Reply:
Sorry. I added that in for my own sake but forgot to remove it since I didn't realize there was a SHOW_PROGRESS global variable.